uiowa/uiowa
# testing adding better js to videos

Resolves https://github.com/uiowa/uiowa/issues/5221
Resolves https://github.com/uiowa/uiowa/issues/5373

currently waiting on approval from band team


## To test
Pull branch, composer install, blt frontend
Sync the homepage
View the homepage
Verify that you can click anywhere on the vertical videos and that you can interact with them like you think you should.
Open browserstack
go to the local homepage on an ipad in both safari and chrome
test that you can interact with the videos as desired there as well. (Note, videos will load slow on ipads, there is potential buffering that will occur)

an accessibility fix for accordions has also been tacked in here, test them at your leisure.
